mlir/lib/CAPI/Dialect/MIGraphX.cpp

```diff
@@ -137,6 +138,10 @@ void mlirMIGraphXAddHighLevelPipeline(MlirPassManager pm) {
   passMan->setNesting(mlir::PassManager::Nesting::Implicit);
   mlir::migraphx::addHighLevelPipeline(*passMan);
   mlir::rock::buildBufferizePipeline(*passMan);
+  if (llvm::sys::Process::GetEnv("MIGRAPHX_MLIR_IR_PRINT")) {
```

Comment:
How is this different from using MIGRAPHX_MLIR_TRACE_MLIR?
If we are doing this change, I d like that documented here.

Reply:
MIGRAPHX_TRACE_MLIR dumps the IR at a couple of selected spots.  This variable enables the mechanism behind --print-ir-after-all, which dumps the IR after each pass.  For me, it was the difference between seeing migraphx.convolution and seeing where tosa.conv3d became rock.conv.

Krzysztof's issue #1227 will make this variable unnecessary by enabling a mechanism to take command-line options through an environment variable.  I'll go ahead and remove this in anticipation.

Reply:
Cool thanks! I was going to suggest this passManager option setting should be done by MIGraphX who owns it.